apps/enc.c

```diff
@@ -579,7 +579,7 @@ static void show_ciphers(const OBJ_NAME *name, void *arg)
     struct doall_enc_ciphers *dec = (struct doall_enc_ciphers *)arg;
     const EVP_CIPHER *cipher;
 
-    if (!islower((unsigned char)*name->name))
+    if (!ossl_islower((unsigned char)*name->name))
```

Comment:
One can probably question if this is actually universally sane condition, but it's probably appropriate to keep it locale-specific. For internally-provided ciphers standard islower works, and remaining question would be ciphers added by engine. [Though I don't really know if they are list-able. But nevertheless dependence on locale seems appropriate in this case.]